config/src/torii.rs

```diff
@@ -62,7 +62,7 @@ pub mod uri {
     /// The web socket uri used to subscribe to blocks stream.
     pub const BLOCKS_STREAM: &str = "block/stream";
     /// Get pending transactions.
-    pub const PENDING_TRANSACTIONS: &str = "pending_transactions";
+    pub const PENDING_TRANSACTION: &str = "pending_transaction";
```

Comment:
With the semantics changed to only return a single matching transaction, I think this endpoint should be renamed to `original_transaction` (open to other name suggestions). `pending_transaction` is very non-descriptive though..

Reply:
As we've kinda returned to how it was before but with a twist, do you think this is no longer applicable?

Reply:
Not sure... It still filters transactions with no way to get all pending transactions, so the concern is still kind of applicable?

Other possible names: `matching_pending_transactions` or just `matching_transactions`